### DISCUSSION TITLE
Compose-only UI

### DISCUSSION THREAD

**Discussion by arashcoder** [Kotlin/kmm-production-sample#24] [General]
Hi! Thanks for this nice sample. I can see that you're using fragments and inside each fragment you're using Composables. I was wondering if there's any reason you decided to go this way and not use Composables directly along with Navigation Component for Compose? Because as we all know, fragments are a major headache in android.
I'd really appreciate your insights, since we're bootstrapping a new project soon and we'd like to base that on this sample.
Thanks!

**Comment by terrakok** [comment]
> fragments are a major headache

I don't agree :)
Of course in the future compose will replace fragments at all.
But at the moment there isn't a good navigation solution for me. I don't like the navigation component.
